py-polars/polars/type_aliases.py

```diff
@@ -69,7 +69,7 @@
 ComparisonOperator: TypeAlias = Literal["eq", "neq", "gt", "lt", "gt_eq", "lt_eq"]
 
 # selector type
-SelectorType: TypeAlias = "_selector_proxy_"
+SelectorType = TypeVar("SelectorType", "Expr", "_selector_proxy_")
```

Comment:
I don't understand this change - these methods first accepted only strings - now they accept Expressions? Can I also pass `pl.col('a')`? I don't think that is what we want, right?

Reply:
> Maybe I am misunderstanding what TypeVar does here.

No, I think I am, hah; was actually trying to more-tightly specify the type to include the inheritance chain, but double-checking it seems I did precisely the opposite - will revert that :)

Reply:
Fixed